sphinx/texinputs/latexmkjarc

```diff
@@ -0,0 +1,7 @@
+$latex = 'platex --halt-on-error --interaction=nonstopmode -kanji=utf8 %O %S';
+$dvipdf = 'dvipdfmx %O -o %D %S';
+$makeindex = 'mendex -U -f -d %B.dic -s python.ist %S || echo "mendex exited with error code $? (ignoring)" && : >> %D';
```

Comment:
On windows, `type NUL >> %D` works fine. But other platform don't.
One possibility is to run a perl script to change the timestamp.